nemo_skills/inference/generate.py

```diff
@@ -265,7 +265,7 @@ def __init__(self, cfg: GenerateSolutionsConfig):
     def setup_llm(self):
         if self.cfg.code_execution:
             sandbox = get_sandbox(**self.cfg.sandbox) if self.cfg.sandbox is not None else None
-            llm = get_code_execution_model(**self.cfg.server, sandbox=sandbox)
+            llm = get_code_execution_model(**self.cfg.server, tokenizer=self.cfg.tokenizer, sandbox=sandbox)
```

Comment:
the full tokenizer should be like this. Maybe we can reassign this somewhere before? It's also used in setup_prompt this way

```python
self.cfg.tokenizer or self.cfg.server["model"]
```

Reply:
Okay, reversed the order of setting prompt